source/particle/property/cpo_elastic_tensor.cc

```diff
@@ -73,7 +73,7 @@ namespace aspect
         AssertThrow(manager.check_plugin_order("crystal preferred orientation","cpo elastic tensor"),
                     ExcMessage("To use the cpo elastic tensor plugin, the cpo plugin needs to be defined before this plugin."));
 
-        cpo_data_position = manager.get_data_info().get_position_by_plugin_index(manager.get_plugin_index_by_name("cpo"));
+        cpo_data_position = manager.get_data_info().get_position_by_plugin_index(manager.get_plugin_index_by_name("crystal preferred orientation"));
```

Comment:
@MFraters: So this was wrong before and we didnt notice because there are no tests? ;-)

Reply:
it prevents people from using it before it is ready :P